package.json

```diff
@@ -12,10 +12,12 @@
     "ethereumjs-util": "4.5.0",
     "fake-merkle-patricia-tree": "^1.0.1",
     "functional-red-black-tree": "^1.0.1",
-    "merkle-patricia-tree": "^2.1.2"
+    "merkle-patricia-tree": "^2.1.2",
+    "safe-buffer": "^5.1.1"
   },
   "devDependencies": {
     "babel-preset-es2015": "^6.24.0",
+    "babel-preset-stage-0": "^6.24.1",
```

Comment:
@axic this is only to build the bundle in the examples, I can build the bundle and not include the dev dependency in `package.json`, since it's examples I think a new dependency isn't needed, what do you say?